File: app/src/main/java/com/anokmik/databinding/fragment/NotifyGreetingFragment.java
package com.anokmik.databinding.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.anokmik.databinding.R;
import com.anokmik.databinding.communicator.NotifyGreetingFragmentCommunicator;
import com.anokmik.databinding.databinding.FragmentNotifyGreetingBinding;

public class NotifyGreetingFragment extends BaseFragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_notify_greeting, container, false);
        FragmentNotifyGreetingBinding binding = FragmentNotifyGreetingBinding.bind(view);
        binding.setCommunicator(new NotifyGreetingFragmentCommunicator());
        return view;
    }

}

File: app/src/main/kotlin/com/anokmik/databinding/fragment/NotifyGreetingFragment.kt
package com.anokmik.databinding.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.anokmik.databinding.R
import com.anokmik.databinding.communicator.NotifyGreetingFragmentCommunicator
import com.anokmik.databinding.databinding.FragmentNotifyGreetingBinding

class NotifyGreetingFragment : BaseFragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater?.inflate(R.layout.fragment_notify_greeting, container, false)
        FragmentNotifyGreetingBinding.bind(view).communicator = NotifyGreetingFragmentCommunicator()
        return view
    }

}